src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -16,30 +16,37 @@ import './WebexLocalMedia.scss';
  * @param {string} props.meetingID  ID of the meeting from which to obtain local media
  * @returns {Object} JSX of the component
  */
-export default function WebexLocalMedia({meetingID}) {
+export default function WebexLocalMedia({className, meetingID}) {
   const [mediaRef, {width}] = useElementDimensions();
   const {localVideo} = useMeeting(meetingID);
   const {ID} = useMe();
   const videoRef = useStream(localVideo);
 
-  const classBaseName = 'local-media';
-  const classObjects = {};
+  const classBaseName = `${WEBEX_COMPONENTS_CLASS_PREFIX}-local-media`;
+  const mainClasses = {
+    [classBaseName]: true,
+    desktop: width >= PHONE_LARGE,
+    'no-media': localVideo === null,
+  };
 
-  classObjects[`${WEBEX_COMPONENTS_CLASS_PREFIX}-${classBaseName}`] = true;
-  classObjects[`${WEBEX_COMPONENTS_CLASS_PREFIX}-${classBaseName}-desktop`] = width >= PHONE_LARGE;
-  classObjects[`${WEBEX_COMPONENTS_CLASS_PREFIX}-no-media`] = localVideo === null;
-
-  const cssClasses = classNames(classObjects);
+  if (className) {
```

Comment:
You can just add this as a last item on `mainClasses`, and there is no need to check it. 
Since `className` defaults to `''`, classNames will computed to false and ignore it, otherwise, whatever value it has will be added as-is, which would be the class name we want to add
See my suggestion.